hw_decorator.py

```diff
@@ -1,15 +1,111 @@
+import random
+from functools import wraps
+
+
 # ЗАДАЧА-1
 # Написать свой декоратор который будет проверять остаток от деления числа 100 на результат работы функции ниже.
 # Если остаток от деления = 0, вывести сообщение "We are OK!», иначе «Bad news guys, we got {}» остаток от деления.
+def division_by_one_hundred(func):
+    """Decorator check the remainder of dividing 100 by the result of the function below """
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        result_func = func(*args, **kwargs)
+        if result_func % 100 == 0:
+            print("We are OK!")
+        else:
+            print(f"Bad news guys, we got {result_func % 100}")
+        return result_func
+
+    return wrapper
+
+
+@division_by_one_hundred
+def random_choice_number():
+    number = random.choice(range(100, 300))
+    return number
+
```

Comment:
If I am not mistaken, the task was to check if 100 can be divided by the number without remainder of the division

Reply:
yes, I misunderstood the task, I will redo it